src/main/perl/Log.pm

```diff
@@ -26,75 +41,70 @@ CAF::Log - Simple class for handling log files
 
   use CAF::Log;
 
-  my $log=CAF::Log->new('/foo/bar','at');
+  my $log = CAF::Log->new('/foo/bar', 'at');
 
   $log->print("this goes to the log file\n");
   $log->close();
 
-=head1 INHERITANCE
-
-  CAF::Reporter
-
 =head1 DESCRIPTION
 
 The B<CAF::Log> class allows to instantiate objects for writing log files.
 A log file line can be prefixed by a time stamp.
 
-
-=over
-
-=cut
-
-#------------------------------------------------------------
-#                      Public Methods/Functions
-#------------------------------------------------------------
-
-=pod
-
-=back
-
 =head2 Public methods
 
 =over 4
 
-=item close(): boolean
+=item C<close()>: boolean
 
-closes the log file.
+closes the log file, returns SUCCESS on succes, undef otherwise
+(if no FH attribute exists).
 
 =cut
 
-sub close ($) {
-  my $self=shift;
+# Called during DESTROY, use $_XYZ flavour
+sub close ($)
+{
+    my $self = shift;
 
-  return undef unless (defined $self->{'FH'});
-# Why adding extra newlines???
-#   $self->{'FH'}->print("\n");
-  $self->{'FH'}->close();
-  $self->{'FH'} = undef;
+    return unless (defined $self->{$_FH});
```

Comment:
its mainly to prevent typos. if this has a typo, perl will not complain.
also, `FH` in your example could be the string `FH` or a constant.
 
the `$_FH` is special due to DESTROY quirks i ran into while testeing this.